### ISSUE TITLE
prettyStack.filter is not a function

### ISSUE THREAD

**Issue by ghasemikasra39** [getsentry/sentry-react-native#846]
OS:
- [ ]  Windows
- [ ]  MacOS
- [ X ]  Linux

Platform:
- [X ]  iOS
- [ X]  Android

SDK:
- [ X]  `@sentry/react-native`
- [ ]  `react-native-sentry`

SDK version: 1.3.7
`react-native` version: 0.62.2

Are you using Expo?
- [ ]  Yes
- [X ]  No

Are you using sentry.io or on-premise?
- [X ]  sentry.io (SaaS)
- [ ]  on-premise

Configuration:

```javascript
Sentry.init({
  dsn: 'MY DSN', 
  debug: true,
  environment: 'development',
});
```

---
I have following issue:
When I run my app, I receive this warning:
```javascript
Sentry Logger [Warn]: Unable to symbolicate stack trace: prettyStack.filter is not a function
```

**Steps to reproduce:**
- Just initials your sentry as above

**Actual result:**
```javascript
Sentry Logger [Warn]: Unable to symbolicate stack trace: prettyStack.filter is not a function
```

**Expected result:**
I should not get any warnings

**Comment by mcavaliere** [comment]
Getting this as well: 

@sentry/react-native: ^1.3.7
react-native: 0.62.2

**Comment by HazAT** [comment]
Fixed in `1.3.8`
